### ISSUE TITLE
No logfile for troubleshooting

### ISSUE THREAD

**Issue by dobby5** [nextcloud/files_frommail#2]
Hello,

I installed the app as described, but it does not work. Also, the log file is not generated to make the error perhaps apparent.

**Comment by daita** [comment]
have you tried to set **debug** to **true** in the NextcloudMailCatcher.php file ? It will generate a log in /tmp/.

Can you describe you current status ?

**Comment by dobby5** [comment]
Yes, I have set it to true. But I haven´t Nextcloud Email Parser.log in temp.

$config = [
	'nextcloud' => 'xx',
	'username'  => 'xx',
	'password'  => 'xx',
	'debug'     => true
];

;)

**Comment by MichaelSmi** [comment]
Push

**Comment by tikoflano** [comment]
@dobby5 What mail server are you using? How did you set up the pipe redirection to the `NextcloudMailCatcher.php` script?

I tried it on a cPanel server with Exim and managed to get it to work (including the logfile) with a few tweaks.

**Comment by daita** [comment]
new version has been released, you can now have some feedback by adding test when manually running the command:

>        $ php -f NextcloudMailCatcher.php test
